CMakeLists.txt

```diff
@@ -282,143 +307,152 @@ set(ARCHIVE_OUTPUT_DIRECTORY ./bin)
+        message(INFO " Used distributed build.")
+    else(CCACHE_FOUND)
+        message(
+            WARNING
+                " distcc was not found. Distributed compilation is impossible."
+)
+    endif(DISTCC_FOUND)
 endif(USE_DISTCC)
 
 if(ENABLE_SANITIZE)
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsanitize=address")
-  message(STATUS "Sanitizing enabled")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsanitize=address")
+    message(STATUS "Sanitizing enabled")
 endif()
 
 if(ENABLE_GCOV)
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} --coverage")
-  add_definitions(-DGCOV_ENABLED)
-  message(STATUS "GCOV enabled")
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} --coverage")
+    add_definitions(-DGCOV_ENABLED)
+    message(STATUS "GCOV enabled")
 endif()
 
 set(CMAKE_CXX_FLAGS_RELEASE " -DNDEBUG -s -O2")
 set(CMAKE_CXX_FLAGS_DEBUG " -DDEBUG -g3 -ggdb3")
 
 if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
-  add_definitions(-DOS_LINUX)
+    add_definitions(-DOS_LINUX)
 elseif(CMAKE_SYSTEM_NAME STREQUAL "QNX")
-  add_definitions(-DOS_QNX)
-  set(BUILD_BT_SUPPORT OFF)
-  set(BUILD_BACKTRACE_SUPPORT OFF)
-  set(EXTENDED_MEDIA_MODE OFF)
+    add_definitions(-DOS_QNX)
+    set(BUILD_BT_SUPPORT OFF)
+    set(BUILD_BACKTRACE_SUPPORT OFF)
+    set(EXTENDED_MEDIA_MODE OFF)
 endif()
 
 if(BUILD_BACKTRACE_SUPPORT)
-  add_definitions(-DBACKTRACE_SUPPORT)
-  message(STATUS "Backtrace support enabled")
+    add_definitions(-DBACKTRACE_SUPPORT)
+    message(STATUS "Backtrace support enabled")
 endif()
 
 if(ENABLE_LOG)
-  add_definitions(-DENABLE_LOG)
-  set(install-3rd_party_logger "install-3rd_party_logger")
-  message(STATUS "Logger enabled")
+    add_definitions(-DENABLE_LOG)
+    set(install-3rd_party_logger "install-3rd_party_logger")
+    message(STATUS "Logger enabled")
 endif()
 
 set(install-bson_c_lib "install-bson_c_lib")
 
 if(TELEMETRY_MONITOR)
-  add_definitions(-DTELEMETRY_MONITOR)
-  message(STATUS "Telemetry monitor enabled")
+    add_definitions(-DTELEMETRY_MONITOR)
+    message(STATUS "Telemetry monitor enabled")
 endif()
 
 add_definitions(-DOS_POSIX)
 
 if(EXTENDED_MEDIA_MODE)
-  add_definitions(-DEXTENDED_MEDIA_MODE)
-  message(STATUS "Extended media mode enabled")
-  # required to find 'glibconfig.h'
-  find_package(PkgConfig)
-  pkg_check_modules(GLIB2 REQUIRED glib-2.0)
-  add_definitions(${GLIB2_CFLAGS})
+    add_definitions(-DEXTENDED_MEDIA_MODE)
+    message(STATUS "Extended media mode enabled")
+    # required to find 'glibconfig.h'
+    find_package(PkgConfig)
+    pkg_check_modules(GLIB2 REQUIRED glib-2.0)
+    add_definitions(${GLIB2_CFLAGS})
 endif()
 
 # Building application
 
 # --- Type HMI
 if(HMI STREQUAL "web")
-  set(WEB_HMI ON)
-  add_definitions(-DWEB_HMI)
+    set(WEB_HMI ON)
+    add_definitions(-DWEB_HMI)
 else()
-  set(HMI "no")
-  add_definitions(-DNO_HMI)
+    set(HMI "no")
+    add_definitions(-DNO_HMI)
 endif()
 
-if(HMI STREQUAL "web" AND NOT DEFINED HMIADAPTER)
-  set(HMIADAPTER "messagebroker")
+if(HMI
```

Comment:
@vilyukh  the old version of this condition looks better

Reply:
@BSolonenko It's depends on "line_width" property. If length string overflow on that value, string will be formatted.

Reply:
@MaximRybak but write one word in one line bad idea